.rubocop.yml

```diff
@@ -5264,7 +5264,7 @@ Style/SymbolLiteral:
 
 Style/SymbolProc:
   Description: 'Use symbols as procs instead of blocks when possible.'
-  Enabled: true
+  Enabled: false
```

Comment:
I think we should leave this as false even though it can throw errors so that we can refactor later.